src/components/Inputs/DepositAmountInput/DepositAmountInput.tsx

```diff
@@ -15,6 +15,8 @@ interface IProps {
   blockerInfo?: string
   decimalsLimit: number
   onBlur?: () => void
+  delta: number
+  dollars: number
```

Comment:
maybe rename this prop to "usdValue" or something like that

Reply:
updated